test/schemas/bookshop-graphql.gql

```diff
@@ -1299,20 +1289,6 @@ input Decimal_filter {
   ne: [Decimal]
 }
 
-"""
```

Comment:
I am not sure why `npm run test:generate-schemas` changed this. May be I should revert it manually ?

Reply:
Yes, could you please revert `bookshop-graphql.gql` entirely? The schema has been adjusted to use the upcoming `minorUnit` from the not yet released cds@6.8.0. (see https://github.com/cap-js/graphql/pull/70)

Reply:
The schema generation test currently skips `bookshop-graphql.gql` for `cds<6.8.0`